front/admin/cli.ts

```diff
@@ -44,18 +46,19 @@ const workspace = async (command: string, args: parseArgs.ParsedArgs) => {
         sId: generateLegacyModelSId(),
         name: args.name,
       });
-      await Promise.all([
-        GroupResource.makeNew({
-          name: "System",
-          type: "system",
-          workspaceId: w.id,
-        }),
-        GroupResource.makeNew({
-          name: "Workspace",
-          type: "global",
-          workspaceId: w.id,
-        }),
-      ]);
+
+      const lightWorkspace = renderLightWorkspaceType({ workspace: w });
```

Comment:
Maybe for later, but this code path should go through the `createWorkspace` function IMHO.

Reply:
yes, it should be in a WorkspaceResource.create